src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Refactoring/RenameEndpoint.cs

```diff
@@ -44,7 +43,8 @@ internal class RenameEndpoint : IRenameEndpoint
             LanguageServerFeatureOptions languageServerFeatureOptions,
             RazorDocumentMappingService documentMappingService,
             ClientNotifierServiceBase languageServer,
-            ILoggerFactory loggerFactory)
+        ILoggerFactory loggerFactory)
```

Comment:
Could be github, but indentation looks off here, and the next line. Seems they both need 4 more spaces?